src/components/apps/launch/index.js

```diff
@@ -36,6 +38,7 @@ const Launch = ({ app, launchError, loading }) => {
             onSuccess: (resp, { onSuccess }) => {
                 router.push(`/${NavigationConstants.ANALYSES}`);
                 onSuccess(resp);
+                trackIntercomEvent(IntercomEvents.LAUNCHED_JOB, resp);
```

Comment:
I see a few of these are sending the response for tracking.  If the response body were large, does this noticeably affect performance?  I'm guessing not, but thought I'd ask

Reply:
I didn't see any noticeable performance issue. And when am tracking listings, I only log the query parameters and not the actual listing so I we are not sending large response bodies to intercom.